partiql-app/partiql-cli/src/main/kotlin/org/partiql/pico/CommandPartiQL.kt

```diff
@@ -12,8 +12,22 @@
  *  language governing permissions and limitations under the License.
  */
 
-package org.partiql.cli
+package org.partiql.pico
```

Comment:
This is deceiving -- I deleted PartiQLCommand because we weren't using it. Then, I created a CommandPartiQL for PicoCLI.